src/footprint/azext_footprint/generated/_help.py

```diff
@@ -0,0 +1,236 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['footprint profile'] = """
+    type: group
+    short-summary: footprint profile
+"""
+
+helps['footprint profile list'] = """
+    type: command
+    short-summary: Retrieves the information about all Footprint profiles under a subscription.
+    examples:
+      - name: List Footprint profiles in a Resource Group
+        text: |-
+               az footprint profile list --resource-group "rgName"
+"""
+
+helps['footprint profile show'] = """
+    type: command
+    short-summary: Retrieves the information about a single Footprint profile.
+    examples:
+      - name: profilesGet
```

Comment:
I saw you already changed the swagger spec. Is it a bug of auto gen or forget to regenerate?

Reply:
needed a regen. good catch!